### PR TITLE
[IMP] mail: move odoobot avatar logic in partner model

### PR DESCRIPTION
**Pull request by xdu-odoo**
task-2348684

### CODE REVIEW

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

File: addons/mail/static/src/models/partner/partner.js [hunk 1]
```diff
@@ -195,6 +195,9 @@ function factory(dependencies) {
          * @returns {string}
          */
         _computeAvatarUrl() {
+            if (this === this.messagingPartnerRoot) {
```
Comment (seb-odoo) [inline addons/mail/static/src/models/partner/partner.js:198]: `this.env.messaging.partnerRoot` would be better I think, we usually don't reference "dependency fields" in compute methods.

Comment (alexkuhn) [reply addons/mail/static/src/models/partner/partner.js:198]: Indeed

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**